arm/proofs/instruction.ml

```diff
@@ -1763,18 +1803,23 @@ let arm_ZIP1_ALT =      REWRITE_RULE all_simd_rules arm_ZIP1;;
 
 let ARM_OPERATION_CLAUSES =
   map (CONV_RULE(TOP_DEPTH_CONV let_CONV) o SPEC_ALL)
+    (*** Alphabetically sorted, new alphabet appears in the next line ***)
```

Comment:
I added this custom rule to avoid introducing bigger diff in the future due to line wrap after adding a new instruction.
What do you think? If this looks ugly, I will revert to the old version.